src/const.ts

```diff
@@ -120,6 +120,16 @@ export interface BotStudioEditProps {
   suggestedRepliesDirection?: 'horizontal' | 'vertical';
 }
 
+export interface MessageInputControlProps {
+  /**
+   * If undefined, false, or 0 is given, message input will not be blocked.
+   * If true or non-zero numeric value is given, message input will be blocked.
+   * If true is given, there will not be a force unblock.
+   * If numeric value is given, it will be used as timeout to force unblock the input.
+   */
+  blockInputWhileBotResponding?: boolean | number;
```

Comment:
This is already an option under input control. Wouldn't it be okay to exclude "input"?
`blockWhileBotResponding`